docs/build/quick_start.md

```diff
@@ -0,0 +1,236 @@
+cmake --build --preset debug
+```
+
+!!! Note
+    By default, configure preset `debug` enables building C++ problem solver tests.
+
+#### Run C++ problem solver tests in Debug
+
+After that, you can go to `build/Debug` and run tests via `ctest`:
+
+```sh
+cd build/Debug
+ctest -V
+```
+
+You can also check code formatting. To learn more, go to the [CMake flags](cmake_flags.md) page.
+
+### Use C++ problem solver in Release
+
+#### Install dependencies with Conan
+
+To install it, run in the root of the project:
+
+```sh
+conan install . -s build_type=Release --build=missing
+```
+
+#### Build C++ problem solver in Release
+
+To build C++ problem solver in release mode using Conan-provided dependencies, run:
+
+```sh
+# release build type without tests
+cmake --preset release-conan
+cmake --build --preset release
+```
+
+To build C++ problem solver with tests in release mode using Conan-provided dependencies, run:
+
+```sh
+# release build type with tests
+cmake --preset release-with-tests-conan
+cmake --build --preset release
+```
+
+#### Run C++ problem solver tests in Release
+
+After that, you can run tests:
+
+```sh
+cd build/Release
+ctest -V
+```
+
+You can also check code formatting. To learn more, go to the [CMake flags](cmake_flags.md) page.
+
+## Start develop Python problem solver of ostis-metasystem
+
+### Install dependencies with pip3
+
+To install dependencies, run:
+
+```sh
+python3 -m venv platform-dependent-components/problem-solver/py/.venv
+source platform-dependent-components/problem-solver/py/.venv/bin/activate
+pip3 install -r platform-dependent-components/problem-solver/py/requirements.txt
+```
+
+### Run Python problem solver tests
+
+Running Python problem solver requires C++ problem solver to be running. Run C++ problem solver:
+
+```sh
+./build/Release/bin/sc-machine -s kb.bin \
+    -e "path/to/ostis-metasystem/lib/extensions;path/to/sc-machine/lib/extensions"
+```
+
+and after that, run Python problem-solver tests in new terminal:
+
+```sh
+source platform-dependent-components/problem-solver/py/.venv/bin/activate && \
+python3 -m unittest discover platform-dependent-components/problem-solver/py
+```
+
+### Run Python problem solver
+
+Run C++ problem solver:
+
+```sh
+./build/Release/bin/sc-machine -s kb.bin \
+    -e "path/to/ostis-metasystem/lib/extensions;path/to/sc-machine/lib/extensions"
+```
+
+and after that, run Python problem-solver in new terminal:
+
+```sh
+source platform-dependent-components/problem-solver/py/.venv/bin/activate && \
+python3 platform-dependent-components/problem-solver/py/server.py
+```
+
+## Start develop interface of ostis-metasystem
+
+### Install dependencies with npm
+
+To install dependencies, go to the `react-sc-web` directory and run:
+
+```sh
+cd platform-dependent-components/interface/react-sc-web
+npm install
+```
+
+### Build interface 
+
+To build interface, run:
+
+```sh
+npm run build
+```
+
+### Run interface
+
+After run:
+
+```sh
+npm run start
```

Comment:
is this expected?
![image](https://github.com/user-attachments/assets/8ef07e01-fbf6-4681-8991-48cd53974ff6)

Reply:
Fixed. Interface migrated to new API

https://github.com/ostis-ai/ostis-ui-lib/pull/61
https://github.com/ostis-ai/react-sc-web/pull/28